### DISCUSSION TITLE
语言提交频道

### DISCUSSION THREAD

**Discussion by NebulaTechs** [win12-online/win12#129] [General]
为了更好的推广此项目，我们要将其翻译成多种语言。但是由于提交次数可能会超过30次，能放在Issues里面并不合适。特意将它放在讨论里，以防止Issues被覆盖。

**Comment by NebulaTechs** [comment]
这周末我会提交英语（尽可能吧，有可能不行）。

**Reply by User782Tec** [reply]
（可以在我们的Teams与我们协作，这样就能更快推出翻译版本啦）

**Comment by User782Tec** [comment]
可是你不需要每次提交都放在Issues里啊（

**Comment by NebulaTechs** [comment]
昨天的language.js打几分啊？
另外你昨天写了吗？

**Reply by User782Tec** [reply]
没写qwq
整体还行，略有格式及翻译问题

**Reply by NebulaTechs** [reply]
QAQ我的手要开始受刑了

**Comment by NebulaTechs** [comment]
基础规划：
- [ ] zh-hans 简体中文（当前）
- [ ] zh-hant 繁体中文
- [ ] en-us 英语（美国）
- [ ] en-uk 英语（英国）
- [ ] de-de 德语（德国）

高级规划：
- [ ] pl-pl 波兰语 （波兰）
- [ ] ar-eg 阿拉伯语（埃及）（实验）
- [ ] zh-hanc 文言文（实验）

**Comment by NebulaTechs** [comment]
文言文版可能要版本10后发布，因为bing翻译：这是开源项目。-- 此开源目也。

**Reply by NebulaTechs** [reply]
。

**Comment by NebulaTechs** [comment]
[language.zip](https://github.com/tjy-gitnub/win12/files/12207181/language.zip)
半成品

**Comment by tjy-gitnub** [comment]
想不到吧，我花了两天，把英文和繁中都翻了（（（

**Comment by Mbilse** [comment]
> 想不到吧，我花了两天，把英文和繁中都翻了（（（

快更新其他语言！
